src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -94,4 +111,5 @@
   </Target>
   <Import Project="..\..\..\Dependencies\PooledObjects\Microsoft.CodeAnalysis.PooledObjects.projitems" Label="Shared" />
   <Import Project="..\..\..\Dependencies\Collections\Microsoft.CodeAnalysis.Collections.projitems" Label="Shared" />
+  <Import Project="$(RepositoryEngineeringDir)targets\PackageDownloadAndReference.targets"/>
```

Comment:
Is there any reason we couldn't put this is the core `Imports.targets`? Imagine we will need to use this in other placse at some point. If it's just to limit impact of change that's fine.

Reply:
I'd do that once we need it elsewhere. Ideally, we'll switch to the new msbuild/nuget feature that implements the real solution and remove this workaround.